test/integration/tests/crud-entity/offline-crud.test.js

```diff
@@ -63,6 +63,25 @@ function testFunc() {
               .catch(done);
           });
 
+          it('findById() should create in the cache, an entity found on the backend, but missing in the cache', (done) => {
```

Comment:
It seems that the test does not fail if the verification utilities.validateEntity(dataStoreType, collectionName, entity); fails.

Reply:
I hadn't noticed it was an async method. Fixed.